README.md

```diff
@@ -1,58 +1,2501 @@
-# TODO
+# IBM Cloud Kubernetes Service open source Ingress controller
 
-### Building
+TODO short description here - do we want to mentin anything about how this is now officially unsupported so it was instead open sourced?
```

Comment:
Yes, we should state that this project can be uses as-is without any support from IBM.  The main purpose of providing this source is to allow users to continue to use this ingress controller after IBM sunsets it from IBM Cloud Kubernetes Service.